data/Jonnie-Dev.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "John Areola",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software engineer - 👨‍💻 Front end developer || UI Designer",
+  "avatar": "https://github.com/Jonnie-Dev.png",
+  "tags": ["frontend", "Open Source", "design", "internship"],
+  "socials": [
+    {
+      "platform": "twitter",
```

Comment:
`socials` follow a new system now.  pasting an example here but please refer to the link mentioned above 
`"socials": [
    { "icon": "FaTwitter", "url": "https://twitter.com/eddiejaoude" },
    { "icon": "FaGithub", "url": "https://github.com/eddiejaoude" },
    { "icon": "FaYoutube", "url": "https://youtube.com/eddiejaoude" }
  ],`